lib/schedule.js

```diff
@@ -73,10 +73,42 @@ function cancelJob(job) {
   return success;
 }
 
+function gracefulShutdown() {
```

Comment:
Wouldn't it be more flexible if it didn't register itself on SIGINT, but instead provided a synchronous method that would gracefully stop schedule, which could be plugged by end user in whatever way they deem necessary, e. g. via https://github.com/mcollina/on-exit-leak-free or by using DI engine teardown mechanism?